pkgs/applications/audio/airwindows/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, lib, fetchFromGitHub, requireFile, unzip, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "airwindows";
+  version = "unstable-2021-12-24";
+
+  src = fetchFromGitHub {
+    owner = "airwindows";
+    repo = "airwindows";
+    rev = "0e28ead5b7dd759ad07ff7bfbd95e1e458e7fcd2";
+    sha256 = "sha256-QUM7dKPcan9g3miZFtaf4oblYVnJ86iJiyOKi70jV/E=";
+  };
+
+  vst-sdk = stdenv.mkDerivation rec {
+    name = "vstsdk366_27_06_2016_build_61";
```

Comment:
```suggestion
    pname = "vstsdk366";
    version = "27_06_2016_build_61";
```

Reply:
To ease the maintenance burden, I copied vst-sdk from oxefmsynth (also used like this in bespokesynth).